packages/sitecore-jss-react/src/utils.test.ts

```diff
@@ -61,4 +66,30 @@ describe('jss-react utils', () => {
       });
     });
   });
+
+  describe('getAttributesString', () => {
+    it('should construct the attributes string correctly', () => {
+      const attributes = {
+        width: '300',
+        className: 'container',
+        height: '200',
+        style: { width: '100%' },
```

Comment:
Can you add, please, another property, like _height_? To have multiple properties for _Object_  test case, to see how it will be concatenated